style.css

```diff
@@ -544,3 +573,252 @@ textarea:focus {
 .vector {
   background-image: url("./images/Vector.png");
 }
+
+/* Media queries */
+
+@media all and (min-width: 768px) {
+  /* Header section */
+
+  .header-img {
+    display: none;
+  }
+
+  .toggle {
+    display: none;
+  }
+
+  .nav-flex {
+    justify-content: center;
+    height: 20%;
+  }
+
+  .menu-list {
+    display: flex;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+    align-items: center;
+    width: 50%;
+  }
+
+  .menu-item {
+    display: inline;
+    width: auto;
+  }
+
+  .menu-item a {
+    text-decoration: none;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    text-align: right;
+    color: #1a2236;
+  }
+
+  /* Headline section */
+
+  .headline-section {
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+    width: 100%;
+  }
+
+  .headline-description {
+    width: 50vw;
+    margin-right: 10%;
+    margin-left: 10%;
+    align-items: flex-start;
+  }
+
+  .description-decoration {
+    display: block;
+    align-self: end;
+    background-image: url("./images/dots.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: relative;
+  }
+
+  .headline-decoration {
+    background-image: url("./images/image-geometry_1.jpg");
+    background-repeat: no-repeat;
+    background-size: center;
+    background-position: relative;
+    width: 50%;
+    padding-bottom: 80%;
+  }
+
+  .item-3 {
+    align-self: flex-start;
+  }
+
+  /* Works section */
+  .grid-works {
+    grid-template-areas:
+      'title project1 project1'
+      'project2 deco-1 project5'
+      'project3 project3 project5'
+      'deco-2 project4 project4';
+  }
+
+  .div-title {
+    grid-area: title;
+  }
+
+  .vertical-align {
+    writing-mode: vertical-lr;
+    transform: rotate(180deg);
+    margin-left: 5%;
+    margin-right: 10%;
+    margin-bottom: 15%;
+  }
+
+  #project1 {
+    grid-area: project1;
+  }
+
+  #project2 {
+    grid-area: project2;
+  }
+
+  #project2 .img-work {
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+  }
+
+  .deco-work-1 {
+    grid-area: deco-1;
+    background-image: url("./images/deco-works-1.PNG");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: relative;
+  }
+
+  #project5 {
+    grid-area: project5;
+  }
+
+  #project6 {
+    grid-area: project5;
+    margin-top: 3%;
+  }
+
+  #project6 .img-work {
+    order: 1;
+  }
+
+  #project3 {
+    grid-area: project3;
+    width: 50vw;
+  }
+
+  .deco-work-2 {
+    grid-area: deco-2;
+    background-image: url("./images/deco-works-2.PNG");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: relative;
+  }
+
+  #project4 {
+    grid-area: project4;
+    margin-left: 20%;
+  }
```

Comment:
Excellent idea to make it start from the center! :tada: 

Let's try to shrink it a little, maybe we could give it a `width: 60%;`?

Reply:
Hello @Kakalanp,
Thanks for your reply, now I have to change project#4 width and delete the property background-position: relative. Please could you tell me if there are no more changes that I have to do to approve?

Reply:
There wasn't :grin:

Congrats! :clinking_glasses: